### PR TITLE
Idea models and controller actions

### PR DESCRIPTION
**Pull request by ussuritiger**
This PR contains all the necessary entities and controller actions to split the task in the team.

### CODE REVIEW

File: src/AppBundle/Entity/Comment.php [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set content
+     *
+     * @param string $content
+     * @return Comment
```
Comment (dennisdegreef) [inline src/AppBundle/Entity/Comment.php]: Will this return a `Comment` instance?

**Comment by piwi91** [comment]
:+1:

File: src/AppBundle/Entity/Comment.php [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    /**
+     * @return boolean
+     */
+    public function isPrivate()
+    {
+        return $this->private;
+    }
+
+    /**
+     * @param $private
```
Comment (dennisdegreef) [inline src/AppBundle/Entity/Comment.php]: What type is this?

**Comment by Joralf** [comment]
LGTM! :+1:

**Comment by ussuritiger** [comment]
@dennisdegreef Incorporated your feedback.

**Comment by dennisdegreef** [comment]
LGTM :+1:

**Pull request merged**